GOVERNANCE.md

```diff
@@ -0,0 +1,112 @@
+# mlpack governance structure (DRAFT)
+
+Revised Oct. 21st, 2019.
+
+## Introduction
+
+mlpack has grown much since its initial inception as a small project out of a
+university research lab.  Now that there are over 150 contributors, it's
+important that we have a clearly defined process for making our decisions and
+organizing ourselves.
+
+This document aims to clarify the governance of mlpack.  This is a living
+document: it may change over time.  The process for making these changes is
+detailed in the "Governance Changes" section.
+
+## Code of Conduct
+
+mlpack aims to be an open and welcoming environment, and as such, we have a
+code of conduct that helps foster this environment.  See
+(here)[https://github.com/mlpack/mlpack/blob/master/CODE_OF_CONDUCT.md] for
+more information.
+
+## Teams & Roles
+
+To keep overhead minimal, mlpack's teams and roles are simple: there is only
+the [Committers](https://github.com/orgs/mlpack/teams/contributors) team, and
+the [NumFOCUS leadership team](TODO:link).
```

Comment:
Don't think NumFOCUS provides such information, but I guess a simple list does work as well.

Reply:
Oh, I was just going to use Github teams to create a NumFOCUS leadership team.  But I was going to wait until we finalized this to create the team, then update the link here right before merge.

Reply:
Github teams is a good idea.